src/domain/generated_address.rs

```diff
@@ -20,15 +21,15 @@ impl GenerateAddressData {
     }
 }
 
-impl GenerateAddressResponse {
-    //construct response
-    pub fn new(
-        resp_message: &str,
-        resp_data: Option<GenerateAddressData>,
-    ) -> GenerateAddressResponse {
-        GenerateAddressResponse {
-            message: resp_message.to_string(),
-            data: resp_data,
-        }
-    }
-}
+// impl GenerateAddressResponse {
```

Comment:
I think you can get rid of the commented code if you have no use of it.

Reply:
Commented code removed. 🙏